metadata.json

```diff
@@ -29,11 +29,11 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.3.0 < 2015.3.0"
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.5.0 < 5.0.0"
```

Comment:
Bumped to 3.5.0 for the reasons listed in the first commit. First version of Puppet with the FileSystem non-instance calls that are used.